src/MergeContext.js

```diff
@@ -869,8 +869,10 @@ class PullRequest {
             this._log("changes requested by " + userRequested.reviewer);
```

Comment:
> Eduard at https://github.com/measurement-factory/anubis/pull/60#issuecomment-1460978061: I doubt we can get rid of usersVoted because of the reviews history that should be tracked. We cannot immediately suspend in the loop upon encountering 'changes_requested' state because it may be stale.

Good point, agreed! I missed the usersVoted.filter() call inside the loop. Please add the following comment above the current `userRequested` declaration to clarify why we are doing this here:

```C++
// The loop could not just Approval.Block() on the first 'changes_requested'
// vote because that vote could have been dismissed later in the loop.
```